lib/welcome.coffee

```diff
@@ -0,0 +1,44 @@
+class Welcome
+
+  @localize: (defW) ->
+    @defW = defW
+    @updateWelcome()  # first time localize
+    atom.workspace.onDidChangeActivePaneItem (item) =>
+      return unless item
+      if item.__proto__.constructor.name is 'WelcomeView'
+        setTimeout(@updateWelcome, 0)
+
+  @updateWelcome: () =>
+    welcomeTab = document.querySelector('.tab[data-type="WelcomeView"]')
+    welcomeEnabled = welcomeTab.className.includes 'active' if welcomeTab
+    return unless welcomeTab && welcomeEnabled
+    welcomeTab.querySelector('.title').textContent = @defW.Welcome.tabTitle
+
+    welcome = document.querySelector('.welcome .welcome-container')
+    return if welcome is null
+    unless welcome.getAttribute('data-localized') is 'true'
+      subtitle = welcome.querySelector('.welcome-header h1.welcome-title')
+      subtitle.innerHTML = subtitle.innerHTML.replace("A hackable text editor for the 21", @defW.Welcome.subtitle.text1)
+      subtitle.querySelector('sup').textContent = @defW.Welcome.subtitle.superscript
+      subtitle.innerHTML = subtitle.innerHTML.replace(" Century", @defW.Welcome.subtitle.text2)
```

Comment:
for Line no.21 to no.23
  1. IMO it's harder to maintain by hard-coding specific strings and replacing them.
  2. it would be nice and efficient by using template string and so you only need setup `innerHTML` once

```coffee
{text1, superscript, text2} = @defW.Welcome.subtitle
subtitle.innerHTML = "#{text1}<sup>#{superscript}</sup>#{text2}"
```